redbot/core/core_commands.py

```diff
@@ -18,7 +18,8 @@
 import traceback
 from pathlib import Path
 from redbot.core import data_manager
-from redbot.core.utils.menus import menu
+from redbot.core.utils.menus import menu, DEFAULT_CONTROLS
```

Comment:
Also DEFAULT_CONTROLS is not imported in master anymore
```suggestion
from redbot.core.utils.menus import menu
```